mozilla-release/browser/components/nsBrowserContentHandler.js

```diff
@@ -551,17 +551,27 @@ nsBrowserContentHandler.prototype = {
     }
 
     var startPage = "";
-    try {
-      // We do not need to get a 'Show home page' option checked
-      // to show a page a user whats to be displayed as a default
-      // when she/he opens a new window / homepage.
-      // That is why we do not test against browser.startup.addFreshTab prop here.
-      // Calling this.startPage will trigger startPage getter defined below.
-      // Services.prefs.getComplexValue will return an object
-      // which contains a user predefined homepage url.
-      startPage = this.startPage.replace("about:home", "resource://cliqz/freshtab/home.html");
-    } catch (e) {
-      Cu.reportError(e);
+
+    // DB-1878
```

Comment:
DB-1878 looks like an internal ticket. It is not very helpful here, since the ticket system is internal only.

Reply:
It's ok to have internal ticket